apps/wistia-videos/package.json

```diff
@@ -7,27 +7,17 @@
     "@contentful/app-sdk": "^4.16.0",
     "@contentful/dam-app-base": "2.0.32",
     "@contentful/f36-components": "^4.70.0",
-    "@contentful/f36-icons": "^4.29.0",
```

Comment:
lots of unused libraries (icons and tokens are also just imported from the main components library I think)